### DISCUSSION TITLE
Best logo?

### DISCUSSION THREAD

**Discussion by becarpenter** [becarpenter/book6#86] [Polls]
We now have a choice of book6 logos:

The original one, quickly cooked up by Brian: https://github.com/becarpenter/book6/blob/main/book6logo-orig.png

The new one, expertly created by Nick: https://github.com/becarpenter/book6/blob/main/book6logo.png

Nick's version is currently in use. Please vote (during January 2024).

**Comment by timchown** [comment]
On 11 Jan 2024, at 02:03, Brian E Carpenter ***@***.***> wrote:

We now have a choice of book6 logos:

The original one, quickly cooked up by Brian: https://github.com/becarpenter/book6/blob/main/book6logo-orig.png

The new one, expertly created by Nick: https://github.com/becarpenter/book6/blob/main/book6logo.png

Ohh, a book on vi, they say you learn a new vi option every day :)

Tim

**Reply by buraglio** [reply]
Ooof, VI didn't even occur to me, even though it is literally my editor of choice (well, VIM)
